references/video_classification/train.py

```diff
@@ -203,6 +203,8 @@ def main(args):
 
     print("Creating model")
     model = torchvision.models.video.__dict__[args.model](pretrained=args.pretrained)
+    if args.output_classes is not None:
```

Comment:
This does not affect anything in the code. You would need to pass `num_classes=output_classes` to the video model for it to work.

Reply:
True, I fixed this in the latest commit.